docparse/find.go

```diff
@@ -414,10 +415,58 @@ func GetReference(prog *Program, context string, isEmbed bool, lookup, filePath
 
 				continue
 			}
+			var p Param
+			// Check if the tag `includable` is present
+			if f.Tag != nil && strings.Contains(f.Tag.Value, "includable:") {
+				// Split the tags to extract the fields necessary
+				t := strings.Split(f.Tag.Value, "includable:")[1][1:]
```

Comment:
There are methods for this already
https://golang.org/pkg/reflect/#StructTag.Get
or
https://golang.org/pkg/reflect/#StructTag.Lookup